### ISSUE TITLE
Fix: a11y - Conflicting value announcements for color input 542

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#364]
## Bug Report

 Announces "#
Label edit
FFFFFF
blank"

## Expected Behavior

Announces "Label edit blank hex value" or announces

"Label edit
FFFFFF". If the FFFFFF value will not be inputed/accepted on validation should announce the first way with descriptive text indicating the input takes a hex value. If it will be inputed/accepted then should announce the second way with the value being displayed and not automatically removed on focus which is jarring.

**Comment by anicholls** [comment]
@jpante Can you clarify the expected behavior? 

---

> will not be inputed/accepted on validation

Does this mean not editable/read only?

>  If the FFFFFF value will not be inputed/accepted on validation should announce the first way with descriptive text indicating the input takes a hex value.

Why would we describe that the input takes a hex value if you can't actually type into it? 

> not automatically removed on focus which is jarring.

What is this referring to?

---

This is my understanding, but correct me where I'm wrong:

If the hex value is not editable, it should announce "Label edit blank {HEX}" (where {HEX} is "hex value" if no value is defined, or the value of the input (e.g. FFFFFF) if it is defined)

If the hex value is editable, it should announce "Label edit {HEX}" (where {HEX} is the value of the input). It should also have descriptive text indicating the input takes a hex value.

Remaining questions: 
- What attribute should that descriptive text go in?
- Still not sure what the focus comment is referring to.

**Comment by jpante** [comment]
@anicholls 
After further discussion with A11y team. the suggested approach is as follows
- Remove the placeholder attribute on input entry, to eliminate the double readout
- add aria-hidden to the # element to ignore it's read out

**Comment by NicholasBoll** [comment]
Fixed by #639
